packages/web-react/src/components/FieldGroup/demo/FieldGroupDefault.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import FieldGroup from '../FieldGroup';
+
+const FieldGroupDefault = () => (
+  <FieldGroup id="FieldGroupDefault" label="Label">
+    <div className="docs-Box">Item</div>
```

Comment:
Do we have the `DocsBox` component? It may be a time for its creation.

Reply:
We have `<DocsBox />` component -> applied everywhere.